tox.ini

```diff
@@ -26,7 +26,7 @@ setenv =
     ZOPE_INTERFACE_STRICT_IRO=1
 commands =
     pip install -U -e .[test]
-    coverage run -p -m unittest discover -s src {posargs}
+    coverage run -p -m unittest discover -s src/zope {posargs}
```

Comment:
Without this change no tests were discoved.
But this change changed the module names for the classes, to I had to adjust the tests.

I think we are using unittest discover here instead of `zope.testrunner` because the latter depends on `zope.interface` itself.

Reply:
I know I have seen this particular issue before, I just don't remember where and what the fix was. It feels wrong if the `__repr__` needs to be updated to something that doesn't match the actual package name anymore.

Reply:
Yeah, it does not feel right, I might fix it using the import trick I used in https://github.com/zopefoundation/zope.interface/pull/344/files#diff-a91536cc6f84a12e565a527e70ed24f6cf2bec70bb095a8daf2c08698b5ad175R62-R64 – do you think this would be a better aproach?

Reply:
Well, they did the job. I really don't have the depth of knowledge of zope.interface to say if they are good or bad.